test/e2e/usb.e2e.js

```diff
@@ -1,4 +1,5 @@
 const capitalize = require('lodash/capitalize');
+const usb = require('particle-usb');
```

Comment:
i'd like to avoid using other mechanisms besides the CLI itself to manipulate the device. how about instead we go with one of the following options:

1. add a `particle usb inspect` command mirroring `particle serial inspect` but reporting device info like `id`, `type`, `platformId`, `firmwareVersion`, `vendorid`, `productid`, and status like `isOpen`, `isInDfuMode`, `.getDeviceMode()`, `isMeshDevice`, `hasModularFirmware`, etc.

2. extend `particle usb list` command output to include flags for `.getDeviceMode()` and `isMeshDevice` like it currently does for `isInDfuMode`. 

with either of those in place, we'd be able to confirm the `particle usb setup-done` state changes using the CLI itself.

what do you think?

Reply:
here's an example of option 2 👉 https://github.com/particle-iot/particle-cli/tree/setup_done/ch46416-with-list

Reply:
The rationale was that simulating user actions on the device (resetting the device, making sure the LED is blinking blue, etc.) is a separate concern, unrelated to what we're testing here, and if there's a more direct way to do it, then why not? I also try to avoid relying on the system under test to validate its own results, if possible.

With that said, you have a lot more vision of how these tests should evolve, so if you feel strongly about this let's merge your changes in :+1:

Reply:
cool - sound good. i'll merge it in now :+1:

i was also thinking it would be nice if `particle usb setup-done` reset the device as well. as it is currently, you run that command and there's no visible change to your device until you reset it which is kind of confusing.

what do you think?

Reply:
Resetting it is probably an overkill, but making it exit the listening mode sounds like a good idea :+1:

Reply:
nice https://github.com/particle-iot/particle-cli/pull/547/commits/829c07b8c84bfee59db3ef937cc673cc245137fd - that works :+1: